moped-editor/src/views/staff/StaffForm.js

```diff
@@ -128,83 +87,34 @@ const StaffForm = ({ editFormData = null, userCognitoId }) => {
   };
 
   const [modalState, setModalState] = useState(initialModalState);
-  const [isApiErrorOpen, setIsApiErrorOpen] = useState(false);
 
   const {
     register,
     handleSubmit,
     errors,
     control,
     setValue,
-    getValues,
-    formState,
+    watch,
```

Comment:
This update gives the confirm modal the latest and greatest values. Using `getValues` was occasionally returning `undefined` so the frontend password validation was failing even when a valid password was entered to reactivate a user.